apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
+    text-align: center;
+  }
+  
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
+  }
+  
+  .App-header {
```

Comment:
@SakinaBoriwala21 the naming convention is not consistent. for eg. `.App-logo` does not go well with other top level names such as `.form`, what do you think?

Better way would be imo. I am not aware of the react conventions so please free to point out if I am missing something. `.app-logo` would be better and consistent as per the project.

Reply:
Yeah, I will change for it to be more consistent.